packages/dds/map/src/test/memory/map.spec.ts

```diff
@@ -43,12 +43,18 @@ describe("SharedMap memory usage", () => {
 		new (class implements IMemoryTestObject {
 			public readonly title = "Create empty map";
 			public readonly minSampleCount = 500;
-
+			public readonly baselineMemoryUsage = 0; // Example: Baseline memory usage in bytes
```

Comment:
Tests with a baseline of 0 are tricky... in practice, the measurements will almost never be a clean 0, they will oscillate around it, and if the baseline is 0, the calculation for allowed deviation is 0, and thus those tests will (in practice) always fail complaining about a regression. I'm not sure if there's a good way to work around that. I think it'd be preferable to throw an error indicating baselines very close to or exactly 0 are unsupported or something along those lines.

Or actually: if we get `allowedDeviation` specified in bytes instead of a percentage directly by the test author, then we can just check that the measurement is within `baseline - allowedDeviation < measurement  < baseline + allowedDeviation`. Yeah, I think I like that the best.